core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/WebViewYouTubePlayer.kt

```diff
@@ -40,6 +41,12 @@ private class YouTubePlayerImpl(private val webView: WebView) : YouTubePlayer {
   override fun setShuffle(shuffle: Boolean) = webView.invoke("setShuffle", shuffle)
   override fun mute() = webView.invoke("mute")
   override fun unMute() = webView.invoke("unMute")
+  override fun isMutedAsync(callback: BooleanCallback) {
+    val bridge = (webView as WebViewYouTubePlayer).getBridge()
```

Comment:
Remove this getter can cast. Instead pass `YouTubePlayerCallbacks` to the constructor of `YouTubePlayerImpl`.

```kotlin
private class YouTubePlayerImpl(
  private val webView: WebView,
  private val callbacks: YouTubePlayerCallbacks
) : YouTubePlayer {
}
```